common/spec/dependabot/shared_helpers_spec.rb

```diff
@@ -526,6 +526,8 @@ def with_git_configured(&block)
         allow(File).to receive(:open).
           with(described_class::GIT_CONFIG_GLOBAL_PATH, anything).
           and_raise(Errno::ENOSPC)
+        allow(FileUtils).to receive(:rm).
+          with(described_class::GIT_CONFIG_GLOBAL_PATH)
```

Comment:
I think I know the answer, but I want to make sure this won't affect how cleanup happens _after_ the spec runs. Are all artifacts eventually removed when you run these locally?

Reply:
I believe this artifact is not even created here because of the stubbing of `File.open` in the previous line?

Reply:
Oh, I get what you mean, whether the stubbing of `FileUtils.rm` affects and possibly prevents deleting other things. Since I'm only stubbing the call when passing a specific parameter, I think it's ok.

Reply:
I'll merge based on the above, but let me know if it made sense.

Reply:
Yep! That makes sense. Thanks 🙇

Reply:
Thanks to you!